binary_heap.py

```diff
@@ -0,0 +1,53 @@
+# _*_ utf=8 _*_
+
+
+class BinHeap(object):
+    """Implement a Binary-Heap."""
+
+    def __init__(self):
+        """Initialize the Binary-Heap."""
+        self.heap = []
+        self.size = 0
```

Comment:
The heap needs to accept an optional iterable as an argument. From the assignment description: "The constructor for your heap should default to creating an empty heap, but allow for creating a populated given an iterable as an input."